app/src/main/res/values/strings.xml

```diff
@@ -449,4 +446,10 @@
   <string name="correct_submitted_answer_with_append">Correct submitted answer: %s</string>
   <string name="incorrect_submitted_answer">Incorrect submitted answer</string>
   <string name="incorrect_submitted_answer_with_append">Incorrect submitted answer: %s</string>
+
+<!--  Help Activity-->
+  <string name="menu_help">Help</string>
+  <!--  FAQ (Help Activity)-->
```

Comment:
We can align everything with same indentation, though FAQ comes under Help, but keeping same indentation is kind of standard format we follow.

Reply:
Okay, sir, I'll recommit by changing the file.